Source/DafnyLanguageServer.Test/GutterStatus/LinearVerificationGutterStatusTester.cs

```diff
@@ -258,6 +258,7 @@ public abstract class LinearVerificationGutterStatusTester : ClientBasedLanguage
     foreach (var (range, inserted) in changes) {
       ApplyChange(ref documentItem, range, inserted);
       traces.AddRange(await GetAllLineVerificationStatuses(documentItem, verificationStatusGutterReceiver, intermediates: intermediates));
+      await Projects.GetLastDocumentAsync(documentItem);
```

Comment:
This works around what I believe is an actual concurrency issue in gutter icon updates.